### PR TITLE
Add RDR related Workloads

### PR DESCRIPTION
**Pull request by prsurve**
- DOC pending

### CODE REVIEW

File: rdr/busybox/deploymnet/mysql.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      labels:
+        workload : mysql
+    spec:
+      containers:
+      - image: quay.io/prsurve/mysql:latest
+        name: mysql
+        env:
+          # Use secret in real usage
+        - name: MYSQL_ROOT_PASSWORD
+          value: password
```
Comment (Avilir) [inline rdr/busybox/deploymnet/mysql.yaml:22]: same as above, replace the password with placeholder

File: rdr/busybox/deploymnet/io_writer_mysql.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        image: quay.io/prsurve/mysql_data_write:latest
+        command: ['/usr/bin/python3', '/script/db.py']
+        imagePullPolicy: Always
+        env:
+         - name: MYSQL_DB_HOST
+           value: mysql
+         - name: MYSQL_DB_USER
+           value: root
+         - name: MYSQL_DB_PASSWORD
+           value: password
```
[the remark this answers is not in the document]
Comment (prsurve) [reply rdr/busybox/deploymnet/io_writer_mysql.yaml:28]: user does not have to be set any values here some values are static and some are dynamically genered

File: rdr/busybox/deploymnet/mysql.yaml (hunk 1 above)
Comment (prsurve) [reply rdr/busybox/deploymnet/mysql.yaml:22]: user does not have to be set any values here some values are static and some are dynamically genered

File: rdr/busybox/deploymnet/io_writer_mysql.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Avilir) [reply rdr/busybox/deploymnet/io_writer_mysql.yaml:28]: passwords can not be stored as plain text on github

File: rdr/busybox/deploymnet/mysql.yaml (hunk 1 above)
Comment (Avilir) [reply rdr/busybox/deploymnet/mysql.yaml:22]: passwords can not be stored as plain text on github

**Review by Avilir — APPROVED** [review]

**Pull request merged**